src/Workspaces/CSharp/Portable/Extensions/ParenthesizedExpressionSyntaxExtensions.cs

```diff
@@ -249,7 +286,9 @@ private static bool RemovalMayIntroduceInterpolationAmbiguity(ParenthesizedExpre
             return false;
         }
 
-        private static bool RemovalChangesAssociation(ParenthesizedExpressionSyntax node, ExpressionSyntax expression, ExpressionSyntax parentExpression, SemanticModel semanticModel)
+        private static bool RemovalChangesAssociation(
```

Comment:
:memo: The unnecessary wrapping of an unchanged line caused this review to take longer than it otherwise needed to.

Reply:
My editor window is most of my screen and already shows 160 chars without problem.  When it goes beyond that, it actually makes the code harder to read with extra scrolling.  I don't have a problem with a soft limit of around that amount so that code can be viewed on one screen without that being arduous on the authors.  i.e. i'm not asking for 80 columns or something so short that people are always running into it :)

Reply:
Yep, I'm not saying you have to change it, but I've decided to point out aspects of the PR which caused the review process to be slower. Similar situations can be avoided in the future to improve PR throughput.

Reply:
On the other hand, it's going to take less time when anyone makes changes here in the future, such as adding a parameter.